qiita_pet/handlers/api_proxy/util.py

```diff
@@ -0,0 +1,15 @@
+from qiita_db.exceptions import QiitaDBUnknownIDError
+from qiita_db.study import Study
+from qiita_db.user import User
+
+
+def check_access(study_id, user_id):
+    try:
+        study = Study(int(study_id))
+    except QiitaDBUnknownIDError:
+        return {'status': 'error',
+                'message': 'Study does not exist'}
+    if not study.has_access(User(user_id)):
+        return {'status': 'error',
+                'message': 'User does not have access to study'}
```

Comment:
@wasade @squirrelo just a comment on this
@ElDeveloper made the same comment in one of my PR and it actually goes against the design of RESTapi. However, we agreed that in interest of time we are going to leave as it was and we are going to update later. The RESTapi that is currently used by the plugins is in a very early development stage and it can still change a bit.

I'm planning to do those changes while I'm away, so we have a more standard RESTapi.